src/main.tsx

```diff
@@ -1,15 +1,18 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+
 import * as Sentry from '@sentry/react';
 import { BrowserTracing } from '@sentry/tracing';
```

Comment:
@sentry/tracing is beeing deprecated. Everything can be used from the sdk specific package (they are re-exported).

```ts
import * as Sentry from "@sentry/react";
Sentry.init(...

new Sentry.BrowserTracing()
 ...
```